scripts/release/publish-build-artifacts.sh

```diff
@@ -30,8 +33,10 @@ publishPackage() {
   repoUrl="https://github.com/angular/${packageRepo}.git"
   repoDir="tmp/${packageRepo}"
 
-  # Create a release of the current repository.
-  $(npm bin)/gulp ${packageName}:build-release:clean
+  if [[ ! ${COMMAND_ARGS} == *--no-build* ]]; then
```

Comment:
This allows us to decide whether the script itself should build or if we want to build the output before on our own.

This allows us to run publishing & deploying in parallel.